app/build.gradle

```
@@ -17,6 +17,13 @@ android {
         vectorDrawables {
             useSupportLibrary true
         }
+
+        defaultConfig {
+            def properties = new Properties()
+            properties.load(project.file('local.properties').newDataInputStream())
```

```suggestion
            properties.load(project.rootProject.file('local.properties').newDataInputStream())
```

rootディレクトリにlocal.propertiesが最初から入っていると思うので、rootProjectの方を参照したほうが自分でファイル作成しなくて良いなと思います

---

rootに生成されるlocal.propertiesに以下のように書かれていたので使わない方が良いのかなと思いました。
> Do not modify this file -- YOUR CHANGES WILL BE ERASED!
しかし、軽く調べてみてもこのファイルの再生成とかについて出てこなかったので実際のところはよくわからないです。
実際、問題なければrootディレクトリのlocal.propertiesに移動させようと思います

---

なるほど！このままで良さそうです 👍